index.html

```diff
@@ -33,6 +33,8 @@
   <script defer src="https://unpkg.com/@popperjs/core@2"></script>
   <script defer src="https://unpkg.com/tippy.js@6"></script>
   <script defer src="https://unpkg.com/clipboard@2"></script>
+  <script defer src="https://unpkg.com/rangy@1"></script>
+  <script defer src="https://unpkg.com/rangy-classapplier@1"></script>
```

Comment:
Easy way to test is import the local versions from this [site](https://css-tricks.com/reveal-a-background-image-upon-text-selection/) and test if the feature works. Then figure out how to import from unpkg.